deploy/materialized_view_report.sql

```diff
@@ -52,8 +52,8 @@ create materialized view ggircs_swrs.report as (
 );
 
 
-create unique index ggircs_swrs_report_primary_key on ggircs_swrs.report (id);
-create index ggircs_swrs_report_history on ggircs_swrs.report (swrs_report_history_id);
+create unique index ggircs_report_primary_key on ggircs_swrs.report (id);
+create index ggircs_report_history on ggircs_swrs.report (swrs_report_history_id);
```

Comment:
We want to keep swrs in the history id since it is derived from the report

Reply:
fixed